src/visual/seriesColor.js

```diff
@@ -48,9 +52,13 @@ export default {
             var dataEach = function (data, idx) {
                 var itemModel = data.getItemModel(idx);
                 var color = itemModel.get(colorAccessPath, true);
+                var borderColor = itemModel.get(borderColorAccessPath, true);
                 if (color != null) {
                     data.setItemVisual(idx, 'color', color);
                 }
+                if (borderColor != null) {
+                    data.setItemVisual(idx, 'borderColor', borderColor);
+                }
             };
 
             return { dataEach: data.hasItemOption ? dataEach : null };
```

Comment:
![image](https://user-images.githubusercontent.com/1956569/65425034-9d027280-de3f-11e9-919a-8b46054e4c9b.png)

If use `scatter`:
If `symbol: emptyCircle`: The color of scatter is different with legend.
If `symbol: circle` `border` set, legend do not show the border.

```js



var option4 = {
    tooltip: {
        trigger: 'axis',
        axisPointer: {
            type: 'shadow'
        }
    },
    legend: {
        // itemStyle: {
        //     borderWidth: 3
        // },
        // selected: {
        //     '联盟广告': false
        // }
    },
    grid: {
        left: '3%',
        right: '4%',
        bottom: '3%',
        containLabel: true
    },
    xAxis: [
        {
            type: 'category',
            data: ['周一', '周二', '周三', '周四', '周五', '周六', '周日']
        }
    ],
    yAxis: [
        {
            type: 'value'
        }
    ],
    series: [
        {
            name: '直接访问',
            type: 'scatter',
            data: [320, 332, 301, 334, 390, 330, 320],
            symbol: 'emptyCircle',
            itemStyle: {
                borderColor: '#5e1',
                borderWidth: 3
            }
        },
        {
            name: '邮件营销',
            type: 'scatter',
            data: [120, 132, 101, 134, 90, 230, 210],
            symbol: 'emptyCircle',
            itemStyle: {
                borderColor: '#ae2',
                borderWidth: 5
            }
        },
        {
            name: '邮件营销2',
            type: 'scatter',
            data: [21, 33, 11, 22, 22, 11, 22],
            symbol: 'circle',
            itemStyle: {
                borderColor: '#912',
                borderWidth: 5
            }
        }
    ]
};
// chart1.setOption(option4);
testHelper.create(echarts, 'plain4', {
    title: [
        '(Legend symbol of line)??',
    ],
    option: option4,
    height: 300
});

```